database/DatabaseExampleSwiftUI/DatabaseExample/Shared/Models/UserViewModel.swift

```diff
@@ -136,30 +132,30 @@ class UserViewModel: ObservableObject {
     }
   }
 
-  func getPosts(tabOpened: Tab) {
-    switch tabOpened {
+  func getPosts(postsType: ContentView.Tab) {
```

Comment:
If we follow MVVM, a ViewModel should not have a dependency on a View. That said, the enum should not be declared in the ContentView namespace. I wonder if `TabView` can work with an enum declared in the global scope? If yes, then the enum should be moved to `Shared/Models`. Also I would prefer it to be renamed as I mentioned before.

Reply:
I see, should I put the enum to Shared/Models/UserViewModel.swift?

Reply:
I would prefer Shared/Models/PostsType.swift (a separate file).